core/src/main/scala/org/apache/spark/ui/exec/ExecutorsTab.scala

```diff
@@ -59,7 +59,9 @@ class ExecutorsListener(storageStatusListener: StorageStatusListener, conf: Spar
   val executorToShuffleRead = HashMap[String, Long]()
   val executorToShuffleWrite = HashMap[String, Long]()
   val executorToLogUrls = HashMap[String, Map[String, String]]()
-  val executorIdToData = HashMap[String, ExecutorUIData]()
+  var executorEvents = new mutable.ListBuffer[SparkListenerEvent]()
+
+  val MAX_EXECUTOR_LIMIT = conf.getInt("spark.ui.timeline.executors.maximum", 1000)
```

Comment:
It's not really executors but tasks right? we already have a property for limiting tasks, spark.ui.timeline.tasks.maximum. It would be reasonable to apply it here?